### ISSUE TITLE
Date conversion doesn't support strict mode

### ISSUE THREAD

**Issue by Ognian** [mongodb-js/extended-json#9]
As per
http://docs.mongodb.org/manual/reference/mongodb-extended-json/#date
there are 2 representations for a Date:
- an ISO-8601 date string (strict mode)
- a number as implemented

The number way is the one implemented, the ISODate is missing.
Any chance to get the strict mode implemented? 

Meanwhile I'll try to use a replacer function in EJSON.stringify ...

**Comment by Ognian** [comment]
EJSON.stringify works with a replacer function
```
EJSON.stringify(result, function replacer(key, value) {
            if (key == '$date') {
                return new Date(value).toISOString();
            }
            return value;
        });
```
BUT parse can't be modified ....
using json2mongo for parsing until this is solved

**Comment by imlucas** [comment]
Thanks for the report.  This fixed in 1.3.0.
